src/models/Movie/parseMovieData.ts

```diff
@@ -1,15 +1,38 @@
-import { MovieSummary } from './type';
+import { Movie, MovieSummary } from './type';
+import { GenreContentfulEntry } from '../Genre/GenreModel';
 
 // TODO: Fix contentful types, take out the any
-export default function parseMovieData(entry: any): MovieSummary {
+
+// Function to parse basic movie data (MovieSummary)
+export function parseMovieSummary(entry: any): MovieSummary {
```

Comment:
This shouldn't be an any. Should be an unknown at worst or a Contentful type from their package.